### ISSUE TITLE
Cufón is inaccessible with Voiceover (built-in Mac screenreader)

### ISSUE THREAD

**Issue by bramd** [sorccu/cufon#79]
It seems that Cufón rendered text is not accessible for the Voiceover screenreader in the Safari webbrowser. Check the headings at http://www.jijmaaktgeschiedenis.nu/ for an inaccessible example. The headings are represented by Voiceover as "image text, image text" etc.
With the Jaws for windows screenreader, this is accessible, so it seems a browser/screenreader specific issue.
